src/components/table.js

```diff
@@ -741,36 +741,56 @@ export const HeaderCell = props => {
   return h(TextCell, _.merge({ style: { fontWeight: 600 } }, props))
 }
 
+const getSortIcon = (sort, field, sortHover, style) => {
+  let sortIcon
+  if (sort?.field === field) {
+    sortIcon = div({
+      style
+    }, [
+      icon(sort.direction === 'asc' ? 'long-arrow-alt-down' : 'long-arrow-alt-up')
+    ])
+  } else if (sortHover === field) {
+    sortIcon = div({
+      style
+    }, [
+      icon('long-arrow-alt-down')
+    ])
+  }
+  return sortIcon
+}
+
 export const Sortable = ({ sort, field, onSort, children }) => {
+  const [sortHover, setSortHover] = useState()
```

Comment:
Would the purpose of this state be clearer with a boolean like `hovered` vs a string?